aws/request.go

```diff
@@ -445,29 +516,57 @@ func (r *Request) Send() error {
 		return err
 	}
 
+	// TODO (jasdel) Refactoring request error handling needs to consider the
+	// difference between API, connection, and SDK behavior errors. This
+	// consideration must include wrapping of underlying error when any SDK
+	// error occurs.
+
```

Comment:
Consider making a Github issue to track this, and put the issue in the comment

Reply:
Thanks I can actually remove this comment. since the Unwrap makes the embedded errors available.

Reply:
Though #74 does cover refactoring the errors. Will update the GitHub issue to ensure the redesign includes that expectation.